src/main/resources/openapiv2.json

```diff
@@ -81,14 +103,23 @@
                 "tags": [
                     "Consumers"
                 ],
-                "description": "Deletes a consumer instance from your Kafka cluster.",
+                "description": "Deletes a consumer instance from your Kafka Bridge instance.",
```

Comment:
Deletes a specified consumer instance from the Kafka Bridge instance that includes it.

Reply:
@PaulRMellor I would be more explicit. The request must to be send etc etc ... because in the way you are using it seems to me that I can send the DELETE request to any bridge instance and it will take care to delete the consumer instance even on another bridge instance (something like an internal communication between bridges).